### PR TITLE
Configure Travis CI for Android & IOS build

### PR DESCRIPTION
**Pull request by olrudenk**
This will setup Travis CI to run Android build and IOS build in each job.
Add names labels to jobs.
Resolves: OLPEDGE-531

Signed-off-by: Oleksandr Rudenko <ext-oleksandr.rudenko@here.com>

### CODE REVIEW

File: scripts/android/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -xe
+ls -la
```
Comment (zhernovs) [inline scripts/android/build.sh]: Looks like forgotten debug. Please remove.

File: .travis.yml [hunk 2]
```diff
@@ -14,12 +15,32 @@ script: $WORKSPACE/scripts/linux/build.sh && $WORKSPACE/scripts/linux/test.sh
 matrix:
   include:
   - compiler: clang
+    name: Linux Build using clang & tests
```
Comment (andescu) [inline .travis.yml]: It is enough to run tests only once.
Please remove tests from clang build and only leave in gcc.

File: .travis.yml [hunk 3]
```diff
@@ -14,12 +15,32 @@ script: $WORKSPACE/scripts/linux/build.sh && $WORKSPACE/scripts/linux/test.sh
...
+    name: OSX Build
     script: $WORKSPACE/scripts/linux/build.sh
+  - os: osx
+    name: IOS Build
+    script: $WORKSPACE/scripts/ios/build.sh  
   - os: windows
+    name: Windows Build
     env: BUILD_TYPE=RelWithDebInfo WORKSPACE=$TRAVIS_BUILD_DIR
-    script: $WORKSPACE/scripts/windows/build.sh
+    script: $WORKSPACE/scripts/windows/build.sh
```
Comment (andescu) [inline .travis.yml]:
```suggestion
    script: $WORKSPACE/scripts/windows/build.sh
```

File: .travis.yml (hunk 2 above)
Comment (olrudenk) [reply .travis.yml]: Done. Test removed from clang

File: .travis.yml (hunk 3 above)
Comment (andescu) [reply .travis.yml]: Doesn't seem to be resolved. Trailing whitespaces still there. Please remove.

**Comment by olrudenk** [comment]
> What about examples - I mean EDGE_SDK_BUILD_EXAMPLES=ON? Do we plan to include them in this build?
Yes, as you know Android build isn't working with examples.

**Pull request merged**